src/components/Preferences/Preferences.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Transition, animated } from 'react-spring'
+import {
+  AppBar,
+  AppView,
+  BREAKPOINTS,
+  ButtonIcon,
+  IconClose,
+  TabBar,
+  Viewport,
+  breakpoint,
+  font,
+  springs,
+} from '@aragon/ui'
+import LocalIdentitiesComponent from './LocalIdentities'
+import { AragonType } from '../../prop-types'
+import {
+  IdentityContext,
+  identityEventTypes,
+} from '../IdentityManager/IdentityManager'
+
+const TABS = ['Network', 'Manage labels']
+const ESCAPE_KEY_CODE = 27
+
+const Preferences = ({ onClose, smallView, wrapper }) => {
+  const { identityEvents$ } = React.useContext(IdentityContext)
+  const [selectedTab, setSelectedTab] = React.useState(1)
+  const [localIdentities, setLocalIdentities] = React.useState({})
+  const handleGetAll = async () => {
+    if (!wrapper) {
+      return
+    }
+    setLocalIdentities(await wrapper.getLocalIdentities())
+  }
+  const handleClearAll = async () => {
+    if (!wrapper) {
+      return
+    }
+    await wrapper.clearLocalIdentities()
+    setLocalIdentities({})
+    identityEvents$.next({ type: identityEventTypes.CLEAR })
+  }
+  const handleModify = (address, data) => {
+    if (!wrapper) {
+      return
+    }
+    wrapper.modifyAddressIdentity(address, data)
+  }
+  const handleImport = async list => {
+    if (!wrapper) {
+      return
+    }
+    setLocalIdentities({})
+    for (const { name, address } of list) {
+      await wrapper.modifyAddressIdentity(address, { name })
+    }
+    setLocalIdentities(await wrapper.getLocalIdentities())
+    identityEvents$.next({ type: identityEventTypes.IMPORT })
+  }
+  const handleKeyUp = e => {
+    if (e.keyCode === ESCAPE_KEY_CODE) {
+      onClose()
+    }
+  }
+  React.useEffect(() => {
+    handleGetAll()
+    window.addEventListener('keyup', handleKeyUp)
+    return () => window.removeEventListener('keyup', handleKeyUp)
+  }, [])
+
+  return (
+    <AppView
+      smallView={smallView}
+      padding={0}
+      appBar={
+        <StyledAppBar>
+          <Title>My Preferences</Title>
+          <StyledButton label="Close" onClick={onClose}>
+            <IconClose />
+          </StyledButton>
+        </StyledAppBar>
+      }
+    >
+      <Section>
+        <TabBar items={TABS} selected={selectedTab} onChange={setSelectedTab} />
+        <Content>
+          {selectedTab === 0 && <ComingSoon />}
```

Comment:
Should we remove this until it gets moved here? It feels a bit weird to have this screen while the feature is available elsewhere in the app:

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/36158/54932858-deab7280-4f1b-11e9-8343-6288a426ca40.png">

Reply:
I asked before starting all of this development and was told it would be ok to put `Coming soon`, I can update  easily.

Reply:
But would it be an issue to comment it for now?

Reply:
ping @aragon/design to ask if we should hide it or not, before doing any change :-)